multisig-ism/src/metadata.sw

```diff
@@ -12,17 +14,54 @@ pub struct MultisigMetadata {
     validators: Vec<EvmAddress>,
 }
 
-// TODO: fix packing
-fn eth_hash(hash: b256) -> b256 {
-    return keccak256(("\x19Ethereum Signed Message:\n32", hash));
-}
+// TODO: properly import
+const B256_BYTE_COUNT: u64 = 32u64;
```

Comment:
Mm could we make these pub in `bytes-extended`? I think because of the `*` import we'd get access to them then